packages/adapter-cloudflare/index.js

```diff
@@ -99,7 +99,20 @@ function get_routes_json(builder, assets, { include = ['/*'], exclude = ['<all>'
 								file === '_redirects'
 							)
 					)
-					.map((file) => `/${file}`);
+					.reduce((prev, file_path) => {
+						const split_path = file_path.split('/');
+						/**
+						 * There is a limit of 100 rules or less. Take advantage of wildcards and generate as few file paths as possible
+						 * https://developers.cloudflare.com/pages/platform/functions/routing/#create-a-_routesjson-file
+						 */
```

Comment:
should use `//` comment style here
```suggestion
						// There's a limit of 100 rules. Take advantage of wildcards and generate as few file paths as possible
						// https://developers.cloudflare.com/pages/platform/functions/routing/#create-a-_routesjson-file
```

Reply:
Thank you!
Updated!!